plugins/backstage-plugin-coder/src/components/CoderWorkspacesCard/WorkspacesList.tsx

```diff
@@ -97,12 +97,12 @@ export const WorkspacesList = ({
       {workspacesQuery.data?.length === 0 && (
         <>
           {emptyState ?? (
-            <Placeholder displayCta>
+            <Placeholder displayCta={Boolean(repoUrl)}>
               {repoUrl ? (
-                <div style={{ textAlign: 'center' }}>
+                <span style={{ display: 'block', textAlign: 'center' }}>
                   No workspaces found for repo
                   <code className={styles.code}>{repoUrl}</code>
-                </div>
+                </span>
```

Comment:
HTML violation that slipped through – divs aren't allowed to be children of paragraphs